include/rtc.h

```diff
@@ -108,6 +111,14 @@ static inline int rtc_set_config(struct device *dev,
 	return api->set_config(dev, cfg);
 }
 
+static inline int rtc_set_time(struct device *dev,
+			       const u32_t seconds)
+{
+	const struct rtc_driver_api *api = dev->driver_api;
+
+	return api->set_time(dev, seconds);
+}
+
```

Comment:
all the above RTC API modification should go in a separate patch first, then the implementation on the driver you want, and finally this patch for newlib. It's much clearer in git history and the perhaps future changes.

Reply:
Even then, it shouldn't take only seconds since epoch; it should take something with higher resolution. Maybe seconds+microseconds like in `struct timeval`.  If at all possible, it should take a u64_t for seconds: 2038 is just around the corner, and for an embedded operating system, who knows for how many years 1.10 will be used for.

Reply:
@lpereira, Agree. u64_t should be used. However, I found the size of time_t in new lib is still 32 bits and new lib changes to use 64 bits recently by this patch: https://sourceware.org/git/gitweb.cgi?p=newlib-cygwin.git;a=commit;h=4de8754bac676fc7f29082ccf8281b1c73ce629e. We probably need to apply this too.

Reply:
Yes, we'll need to update newlib eventually.

Reply:
Variables storing RTC values are often defined as signed to allow 'going back in time' and to facilitate signed arithmetic. Should we do the same?